### DISCUSSION TITLE
Unable to download/export CSVs from SQL interface

### DISCUSSION THREAD

**Discussion by ryanperronevocal** [sheltermanager/asm3#1589] [Q&A / Help]
Hello. I am needing to download three specific reports from the SQL interface, each time it gives me a Network Error message. I have done trouble-shooting with our network, there are no issues with it. We have attempted to download it from different computers and even from our own home networks on personal computers. We get a network error message every time. We are using an opensource version of ASM. We specifically need to export the "CSV of animal/adopter data', "CSV of person data" and "CSV of animal/medical data" from the SQL interface. Please let me know how to figure what the issue is and fix it! Thanks!

**Comment by bobintetley** [comment]
Hi, is your install sitting behind a reverse proxy with a timeout configured maybe? 

Are there any error messages in the log after trying to export these files? (Where to log is configured in your asm3.conf file. The default is to the syslogger)
